source/gcp/supported-services/vpc_flow.rst

```diff
@@ -0,0 +1,31 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+.. _gcp_vpc_flow:
+
+VPC Flow logs
+=============
+
+`VPC Flow Logs <https://cloud.google.com/vpc/docs/flow-logs>`__ records a sample of network flows sent from and received by VM instances, including instances used as Google Kubernetes Engine nodes. Flow logs are aggregated by connection from Compute Engine VMs and exported in real time.
```

Comment:
```suggestion
`VPC Flow Logs <https://cloud.google.com/vpc/docs/flow-logs>`__ record a sample of network flows sent from and received by VM instances, including instances used as Google Kubernetes Engine nodes. Flow logs are aggregated by connection from Compute Engine VMs and exported in real time.
```

Reply:
In this case "VPC Flow Logs" is the whole name of the service, being "Logs" part of its name, so using the singular is the right choice here.